src/sentry/demo/data_population.py

```diff
@@ -1081,45 +1117,59 @@ def populate_generic_transaction(
 
         self.log_info("populate_generic_error.finished")
 
-    def populate_sessions(self, project, error_file):
+    def populate_sessions(self, project, distribution_fn_num: int, mobile: bool, error_file=None):
         self.log_info("populate_sessions.start")
+
         dsn = ProjectKey.objects.get(project=project)
 
-        react_error = get_event_from_file(error_file)
+        if error_file:
+            error = get_event_from_file(error_file)
 
-        for (timestamp, day) in self.iter_timestamps(4):
+        # keep track of versions for mobile
+        seen_versions = []
+        num_versions = 0
+        weights = []
+
+        for (timestamp, day) in self.iter_timestamps(distribution_fn_num):
             transaction_user = self.generate_user()
             sid = uuid4().hex
             release = get_release_from_time(project.organization_id, timestamp)
             version = release.version
 
-            # initialize the session
+            # add new version if necessary
+            if version not in seen_versions:
+                seen_versions.append(version)
+                num_versions += 1
+                if weights:
+                    weights.append(weights[-1] * random.uniform(2, 2.5))
+                else:
+                    weights = [1]
+
+            self.send_aggr_session(
```

Comment:
@roggenkemper just to be safe, can you add a setting for disabling aggregate settings? Like `DISABLE_AGGREGATE_SESSIONS`? That way we can turn it off if it's a problem in prod.

Reply:
added!